cmd/qat_plugin/dpdkdrv/dpdkdrv_test.go

```diff
@@ -63,6 +63,49 @@ func createTestFiles(prefix string, dirs []string, files map[string][]byte, syml
 	return nil
 }
 
+func TestNewDevicePlugin(t *testing.T) {
+
+	tcases := []struct {
+		name            string
+		dpdkDriver      string
+		kernelVfDrivers string
+		expectedErr     bool
+	}{
+		{
+			name:            "Wrong dpdkDriver",
+			dpdkDriver:      "uio",
+			kernelVfDrivers: "c6xxvf",
+			expectedErr:     true,
+		},
+		{
+			name:            "Correct dpdkDriver, wrong kernelVfDrivers",
+			dpdkDriver:      "vfio-pci",
+			kernelVfDrivers: "c6xxxvf",
+			expectedErr:     true,
+		},
+		{
+			name:            "Correct dpdkDriver, kernelVfDrivers separated wrong",
+			dpdkDriver:      "vfio-pci",
+			kernelVfDrivers: "c6xxvf:d15xxvf",
+			expectedErr:     true,
+		},
+		{
+			name:            "No errors",
+			dpdkDriver:      "vfio-pci",
+			kernelVfDrivers: "c6xxvf,d15xxvf",
+			expectedErr:     false,
+		},
+	}
+	for _, tt := range tcases {
+
+		_, err := NewDevicePlugin(1, tt.kernelVfDrivers, tt.dpdkDriver)
+
+		if tt.expectedErr && err == nil {
+			t.Errorf("Test case '%s': expected error, but got success", tt.name)
+		}
```

Comment:
Also we should check if `err != nil` and we expect no error.

Reply:
yes, of course

Reply:
fixed